mibs/bdcom/NMS-IF-MIB.my

```diff
@@ -0,0 +1,1014 @@
+NMS-IF-MIB  DEFINITIONS ::= BEGIN

```

Comment:
Could you please rename the file to NMS-IF-MIB (remove extension) ?

Reply:
Other that this, looks good @freddy36

Reply:
fixed